docs/proposals/tool-to-backfill-tsdb-blocks.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Tools to backfill TSDB blocks"
+linkTitle:
+weight: 1
+slug:
+---
+
+- Author: [Siddharth Asthana](https://github.com/edith007)
+- Date: March 2023
+- Status: draft
+
+## Overview
+
+There might be a lot of cases where the users might already be using Prometheus and later decides to switch to Cortex. Cortex will take care of the metrics which will be generated after the switch, but at present there is no way to backfill the old Prometheus TSDB blocks to Cortex. This proposal is to add the ability in cortex to fetch the Prometheus TSDB blocks and backfill the data.
+
+## Detailed Solution
+
+We can create a new service called `Importer` which will be responsible for bringing in the old Prometheus TSDB blocks to Cortex.
+
+### How Importer will work
+
+The importer service will have one API endpoints: `import`.
+
+`import`:
+
+- It will take a link to where the old Prometheus TSDB blocks are stored
+- Cortex will download this block from the link received and upload to the long term storage
+
+### How will Cortex import the old TSDB blocks
+
+Cortex will import the old TSDB blocks by using the `cortex-cli` tool, which will be used by
```

Comment:
I think `cortex-cli` tool is still not very clear to me. Is it a separate repo or part of the Cortex repo.

Reply:
Can we make it part of https://github.com/cortexproject/cortex-tools? Users are still using that